test/residency/prototypes/test_task014_durable_process_public_seam.py

```
@@ -0,0 +1,947 @@
+    return (int)syscall(SYS_fsync, file);
+}
+"""
+    with tempfile.TemporaryDirectory(
+        prefix="residency-task014-adversary-"
+    ) as directory:
+        root = Path(directory)
+        executable = root / "task014"
+        interceptor = root / "directory_fsync_einval.c"
+        library = root / "directory_fsync_einval.so"
+        interceptor.write_text(interceptor_source, encoding="utf-8")
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)